docs/source/markdown/podman-run.1.md

```diff
@@ -281,7 +281,7 @@ it in the **containers.conf** file: see **containers.conf(5)** for more informat
 
 #### **--detach-keys**=*sequence*
 
-Specify the key sequence for detaching a container. Format is a single character `[a-Z]` or one or more `ctrl-<value>` characters where `<value>` is one of: `a-z`, `@`, `^`, `[`, `,` or `_`. Specifying "" will disable this feature. The default is *ctrl-p,ctrl-q*.
+Specify the key sequence for detaching a container. Format is a single character `[a-Z]` or one or more `ctrl-<value>` characters where `<value>` is one of: `a-z`, `@`, `^`, `[`, `,` or `_`. Specifying "" will disable this feature.
```

Comment:
```suggestion
Specify the key sequence for detaching a container. Format is a single character `[a-Z]` or one or more `ctrl-<value>` characters where `<value>` is one of: `a-z`, `@`, `^`, `[`, `,` or `_`. Specifying "" will set the sequence to the default value of *ctrl-p,ctrl-q*
```

Reply:
The proposed wording `Specifying "" will set the sequence to the default value of *ctrl-p,ctrl-q*` does change the current meaning.  I have updated the patch with the new meaning.